daemon/lokinet.cpp

```diff
@@ -82,9 +84,6 @@ install_win32_daemon()
     llarp::LogError("Cannot install service ", GetLastError());
     return;
   }
-  // just put the flag here. we eat it later on and specify the
-  // config path in the daemon entry point
-  StringCchCat(szPath.data(), 1024, " --win32-daemon");
```

Comment:
This seems wrong.  If removed here it seems there is nowhere that sets the service up with the `--win32-daemon` flag, but main() still relies on it?